Source/WebCore/platform/encryptedmedia/clearkey/CDMClearKey.cpp

```diff
@@ -442,6 +442,20 @@ RefPtr<CDMInstanceSession> CDMInstanceClearKey::createSession()
     return adoptRef(new CDMInstanceSessionClearKey(*this));
 }
 
+auto CDMInstanceClearKey::getKeyHandleValue(const KeyIDType& keyID) const -> Ref<KeyHandleValuePromise>
```

Comment:
it is very confusing to have CDMInstanceClearKey::getHandleValue and CDMProxy::getHandleValue) return both a `KeyHandleValuePromise` which are actually of different type. 
`Vector<uint8_t>` vs a variant, they should be named differently for readability.